### ISSUE TITLE
Block-nested loop

### ISSUE THREAD

**Issue by kostja** [tarantool/graphql.0#21]
Change the built-in graphql execution algorithm (the one which works with "accessors") from simple nested loop to block-nested loop.
https://en.wikipedia.org/wiki/Block_nested_loop

**Comment by Totktonada** [comment]
After #98 we have breadth-first search executor, which reduces count of network requests (it fetches tuples in blocks), but don't changes execution order, so we need to store the entire data until the end of a request. I filed #236 about that.

**Comment by Totktonada** [comment]
I'm going to archive the repository. I'll proceed as follows:

* Mark all open pull requests with the `archived` label and close.
* Mark all open issues with `archived` and `wontfix` labels and close.
* Archive the repository.

Consider the following projects:

* https://github.com/tarantool/graphql
* https://github.com/tarantool/graphqlapi
* https://github.com/tarantool/graphqlide
